File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c++/ronnyg2121.cpp
#include <iostream>
using namespace std;

int main(int argc, char const *argv[])
{
    // Solución al ejercicio #0
    for (int i = 0; i < 101; i++) {
        if (i %3 == 0 && i %5 == 0) {
            cout << "fizzbuzz\n" << endl;
        }

        else if (i %3 == 0) {
            cout << "fizz\n" << endl;
        }

        else if(i %5 == 0) {
            cout << "buzz\n" << endl;
        }

        else {
            cout << i << endl;
        }        
    }
    
    return 0;
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/ronnyg2121.py
# Mi solución del ejercicio1

for i in range(1, 101, 1):
    if i %3 == 0 and i %5 == 0:
        print("fizzbuzz\n")

    elif i %3 == 0:
        print("fizz\n")
    elif i %5 == 0:
        print("buzz\n")
    else:
        print(f"{i}\n")